NaReguaApp/src/routes/app-prestator.routes.js

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import InicioPrestador from '../paginas/inicio_prestador';
+import CadastroServico from '../paginas/cadastro_servico';
+
+const PrestadorStack = createNativeStackNavigator();
+
+function AppPrestadorRoute() {
+  return (
+    <PrestadorStack.Navigator initialRouteName="InicioPrestador">
+      <PrestadorStack.Screen name="InicioPrestador" component={InicioPrestador} options={{ headerShown: false }}/>
+      <PrestadorStack.Screen name="CadastroServico" component={CadastroServico} options={{ title: 'Novo Serviço'}}/>
```

Comment:
Codacy has a fix for the issue: [Replace `·name="CadastroServico"·component={CadastroServico}·options={{·title:·'Novo·Serviço'}}` with `⏎········name="CadastroServico"⏎········component={CadastroServico}⏎········options={{title:·'Novo·Serviço'}}⏎······`](https://app.codacy.com/gh/UFAPE-Projetao20202/naregua_front/pullRequest?prid=8386487)
```suggestion
      <PrestadorStack.Screen
```